bedrock/mozorg/templates/mozorg/contribute/contribute-tasks.html

```diff
@@ -0,0 +1,182 @@
+              <span class="label-text">{{ _('Download our Stumbler App') }}</span>
+            </li>
+            <li class="option">
+              <a data-task-id="4" class="ab-task category-coding" href="http://mozilla.github.io/webmaker-launch/activity.html" target="_blank"></a>
+              <span class="label-text">{{ _('Test a mobile app!') }}</span>
+            </li>
+          </ul>
+        </div>
+        <div class="option-group">
+          <h4>{{ _('Try something more challenging.') }}</h4>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="1" class="ab-task category-nightly" href="https://nightly.mozilla.org/" target="_blank"></a>
+              <span class="label-text">{{ _('Help us test by downloading Firefox Nightly') }}</span>
+            </li>
+            <li class="option">
+              <a data-task-id="7" class="ab-task category-testing" href="https://oneanddone.mozilla.org/" target="_blank"></a>
+              <span class="label-text">{{ _('Pick from a list of first-time tasks here') }}</span>
+            </li>
+            <li class="option">
+              <a data-task-id="5" class="ab-task category-helping" href="https://support.mozilla.org/en-US/get-involved/questions" target="_blank"></a>
+              <span class="label-text">{{ _('Help users by answering a support question.') }}'</span>
+            </li>
+            <li class="option">
+              &nbsp;
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+  {% else %}
+    <section class="section inquiry">
+      <div class="container inquiry-choices">
+        <div class="option-group">
+          <h4>{{ _('Try something simple.') }}</h4>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="6" class="ab-task category-video" href="https://air.mozilla.org/channels/project-meeting/" target="_blank"></a>
+              <span class="label-text">{{ _('Attend a Mozilla community meeting right now!') }}</span>
+            </li>
+            <li class="option">
+              &nbsp;
+            </li>
+          </ul>
+        </div>
+        <div class="option-group">
+          <h4>{{ _('Try something more challenging.') }}</h4>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="5" class="ab-task category-helping" href="https://support.mozilla.org/en-US/get-involved/questions" target="_blank"></a>
+              <span class="label-text">{{ _('Help users by answering a support question.') }}</span>
+            </li>
+            <li class="option">
+              &nbsp;
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+  {% endif %}
+
+  <section class="section inquiry">
+    <div class="container center">
+      <a href="{{ url('mozorg.contribute.tasksthankyou') }}" class="tasks-more">{{ _('Not ready to try a task?') }} »</a>
```

Comment:
You can use the <code>button-flat</code> class to get the standard button styles, but you may still need another style hook to increase the font size.